Modules/quote/Controller/QuotelistController.php

```diff
@@ -241,13 +283,23 @@ public function editnewuserAction($id_space, $id) {
         $form->setButtonsWidth(2, 10);
         $form->setValidationButton(CoreTranslator::Save($lang), "quotenew/" . $id_space . "/" . $id);
         if ($form->check()) {
-            $id = $modelQuote->set($id, $id_space, $form->getParameter('recipient'),
+            $id = $modelQuote->set(
+                    $id,
+                    $id_space,
+                    $form->getParameter('recipient'),
+                    $form->getParameter('recipient_email'),
                     $form->getParameter('address'),
-                    $form->getParameter('id_belonging'), 0,
+                    $form->getParameter('id_pricing'),
+                    0,
+                    $form->getParameter('id_client'),
                     $form->getParameter('date_open')
             );
             $_SESSION['message'] = QuoteTranslator::QuoteHasBeenSaved($lang);
-            $this->redirect("quotenew/" . $id_space . "/" . $id);
+            if ($id > 0) {
+                $this->redirect("quotenew/" . $id_space . "/" . $id);
```

Comment:
may be fine, but a quotenew route with an id ???  id=0 for a new else edit ?

Reply:
quotenew/{id_space}/{id_quote} precisely leads to a quote edit. But I could change the route name...